openfisca_core/variables.py

```diff
@@ -34,7 +34,7 @@ def get_introspection_data(self, tax_benefit_system):
         return comments, source_file_path, source_code, start_line_number
 
     def to_column(self, tax_benefit_system):
-        formula_class = self.__class__.formula_class
+        formula_class = self.__class__.formula_class  # Formula
```

Comment:
Since you simplified everything, `formula_class` is always `Formula` !

We should: 
- remove the class attribute `formula_class = Formula`
- remove `formula_class` for `new_filled_column` arguments as it's always the same.